docs/design.md

```diff
@@ -56,6 +57,8 @@ details. `devdax` mode is not supported. Though a
 raw block volume would be useful when a filesystem isn't needed, Kubernetes
 cannot handle [binding a character device to a loop device](https://github.com/kubernetes/kubernetes/blob/7c87b5fb55ca096c007c8739d4657a5a4e29fb09/pkg/volume/util/util.go#L531-L534).
 
+<sup>4 </sup> **Huge pages supported by mkfs options** means that filesystem is created using options supporting huge pages. See [Using Persistent Memory Devices with the Linux Device Mapper](https://pmem.io/2018/05/15/using_persistent_memory_devices_with_the_linux_device_mapper.html). The options supporting huge pages are used in both modes, but page faults related to huge pages occur in the direct mode and not in the LVM mode.
```

Comment:
Can you make the link more specific or describe what I should look for in that page? It's not immediately obvious (probably the mkfs parameters and `xfs_io -c "extsize 2m"`.

Reply:
I would rewrite this as:
```
**Huge pages supported**: ext4 and XFS filesystems are created using the options that should enable huge page support, as explained in ["Using Persistent Memory Devices with the Linux Device Mapper"](https://pmem.io/2018/05/15/using_persistent_memory_devices_with_the_linux_device_mapper.html). [Testing that support by observing page faults](/test/cmd/pmem-access-hugepages/main.go) confirmed that this worked for LVM mode. It did not work for direct mode in the QEMU virtual machines, but it cannot be ruled out that it works elsewhere.
```

And then turn "no" into "maybe" in the table...

Reply:
Please preserve line breaks. A full paragraph in a single line makes it harder to comment on a change future diffs will be larger.